### PR TITLE
Purge using namespace std in libyul

### PR DESCRIPTION
**Pull request by junaire**
Related: https://github.com/ethereum/solidity/issues/14403

### CODE REVIEW

**Comment by junaire** [comment]
CC @nikola-matic

File: libyul/AsmAnalysis.cpp [hunk 1]
```diff
@@ -43,16 +43,15 @@
 
 #include <memory>
 #include <functional>
-
-using namespace std;
+using namespace std::string_literals;
```
Comment (nikola-matic) [inline libyul/AsmAnalysis.cpp:47]:
```suggestion

using namespace std::string_literals;
```

**Review by nikola-matic — COMMENTED** [review]
Put back the newline, and we can merge :)

File: libyul/AsmAnalysis.cpp (hunk 1 above)
Comment (junaire) [reply libyul/AsmAnalysis.cpp:47]: Thanks, done.

**Comment by junaire** [comment]
Looks like there are some glitches in the CI, we probably need to retrigger the tests.

**Review by nikola-matic — APPROVED** [review]

**Pull request merged**